### ISSUE TITLE
Questions about finetune

### ISSUE THREAD

**Issue by woxue** [qyxqyx/AIM_FAS#4]
Thanks for code sharing!
I have some detail questions about finetune step in fairly comparison part. The discription in your paper is as follows:"To evaluate its 1-shot perfor- mance, we first finetune it on the support set of the 1-shot tasks and then evaluate it on the corresponding query set." and "We generate 100,000 training tasks on the train set and 100 (T = 100) testing tasks on the test set. "
So I want to know the finetune step in other methods (Resnet-10, FAS-DR, DTN) is jointly (use the testing tasks data in one time) or seperately(use one test task per time and repeat 100 times then compute an average)?

**Comment by qyxqyx** [comment]
Sorry for delayed reply. Thanks for your question. In the few-shot testing scene, we follow the standard few-shot testing manner to evaluate all models. That is, each model should be evaluated on lots of (100 in our work) few-shot learning tasks. on each task, the model first learning on few data (support set) and then evaluating on the other data (query set).

**Comment by woxue** [comment]
Thank you.
